.travis.yml

```diff
@@ -17,14 +16,36 @@ env:
     - DB=MySQLi
     - DB=SQLite
 #    - DB=PostgreSQL
+
 matrix:
+  fast_finish: true
+  include:
+    - php: hhvm
+      sudo: true
+      dist: trusty
+      group: edge # use until the next Travis Image update
+      addons:
+        apt:
+          packages:
+            - mysql-server-5.6
+            - mysql-client-core-5.6
+            - mysql-client-5.6
+      env: DB=MySQLi
+    - php: hhvm
+      sudo: true
+      dist: trusty
+      group: edge # use until the next Travis Image update
+      env: DB=SQLite
   allow_failures:
     - php: nightly
-# Travis have too old HHVM, can't rely on it yet
     - php: hhvm
 #    - env: DB=PostgreSQL
+
 before_script:
-  - mysql -e 'CREATE DATABASE `travis`;'
+  - |
```

Comment:
Well, for compatibility with tests `travis` user should be created, since it is hardcoded in tests.
And this command should be different for Precise and Trusty.